packages/w3c-vc/src/lib/w3c-vc.test.ts

```
@@ -119,6 +122,68 @@ describe('W3C Verifiable Credentials', () => {
+        expect(didKeyPairs.id).toBe(parsed.verificationMethodId);
+        expect(didKeyPairs.controller).toBe(did);
+
+        // Build a credential issued by the new did:key.
+        const credential = {
+          ...modernCredentialV2_0,
+          issuer: did,
+          validFrom: '2024-04-01T12:19:52Z',
+        };
+
+        // Sign with the freshly-generated keypair.
+        const signed = await signCredential(credential, didKeyPairs, cryptosuite as CryptoSuiteName);
```

ESLint

[error] Replace `credential,·didKeyPairs,·cryptosuite·as·CryptoSuiteName` with `⏎··········credential,⏎··········didKeyPairs,⏎··········cryptosuite·as·CryptoSuiteName,⏎········`

(prettier/prettier)